Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -243,20 +249,88 @@ In calculating the SCI value, you SHOULD use the highest granularity data availa
+| Product Name, entity name, or Software Product | metadata/name | MUST | Text | |
+| Contact name | metadata/contact-name | MUST | Text | The point of contact responsible and accountable for the report. |
+| Contact email | metadata/contact-email | MUST | Text | The point of contact responsible and accountable for the report. |
+| Organization | metadata/organization | SHOULD | Text | This SHALL be populated where the report is not by an individual contributor, otherwise it is not required. |
+| GUID | metadata/guid | MUST | GUID | Following a format in [RFC4122], provided to uniquely identify this particular product, resource, or service. |
+| Version | metadata/version | MAY | Text | |
+| Date of Calculation | metadata/calculation-date | MUST | Date | Following a format described in [RFC3339], [a subset](https://ijmacd.github.io/rfc3339-iso8601/) of ISO 8601 |
+| Further information on calculation | metadata/calculation-information | MAY | Text | More information on your calculation methodology can be provided as freetext, or as a URL to an external document or software repository. |
+| Further information on report | metadata/report-information-url | MAY | URL | More information on your calculation methodology may be provided, and this SHALL be a URL to an external document or software repository.  |
+
+
+<!--
+
+- Supporting infrastructure and systems contained within the [Software Boundary](#software-boundary)
+-
+  - `CI` MUST be reported
+  - `C` SHOULD be reported, but where it is not reported, a reason MUST be provided
+  - The `R` baseline used MUST be reported
+  - If the `R` baseline used wa not selected from the  for baselines, then a reason for that choice MUST be provided
+- Metadata for the SCI value being reported
+  - Product name MUST be reported
+  - Contact details, including a name and email, for a point of contact responsible for the report MUST be provided
+  - Organization SHOULD be provided, where the report is not by an individual contributor
+  - A GUID, following a format described in [RFC4122], MUST be provided to uniquely identify this particular product, resource, or service.
+  - A software version for the software product about which this report is made, MAY be provided.
+  - The date the calculation was made, following a format described in [RFC3339], MUST be provided.
+  - More information on your calculation methodology MAY be provided, as a freetext comment, limited to 300 characters.
+  - More information on your report MAY be provided, as a URL linking to an external document or software repository.
+
+
+-->
+
+### Data format for reporting
+The format of your report MUST be machine-readable, and SHOULD be in a commonly used format. Examples of commonly used formats include YAML and JSON. Other formats can be selected, but they MUST allow for the same identifiers to be parsed.
+
+Examples of valid machine-readable submissions can be found in Appendix A.
```

Comment:
Possible we need a link to to the published URL for appendix A here

Reply:
I left another comment but after thinking about this more we need to stick to one format.